NEWS: Add Adibou1 announcement (#313)
scummvm/scummvm-web: data/en/news/20230715.markdown

---
title: A New World of Discoveries
author: sdelamarre
date: 1689415200
---
_Adibou_, the first game in the beloved children's series, is now ready for public testing!

> Also known as _Adi Jr.'s World of Discoveries_, this 1991 Coktel Vision title began the educational Adibou collection aimed at children of age 4 to 7 years.

Reviewer: "Collection" works here, great idea!

> Many of the iconic activities and mechanisms of the series were already part of this first opus: planting flowers, playing the game Breakout, and being rewarded with sweets upon completing Read/Count exercises. And of course, this was Adibou's first appearance! Although at first he could only speak the same strange gibberish as heard in the [Goblins series](https://wiki.scummvm.org/index.php/Goblins_series), he is still the patient and funny learning partner that he's remembered for being in later games.

Reviewer: It's still not clear to me what "at first" means. First game, first release of the first game, or something else. Since you said that it refers to the floppy releases, maybe something like this would work?

```suggestion
Many of the iconic activities and mechanisms of the series were already part of this first opus: planting flowers, playing the game Breakout, and being rewarded with sweets upon completing Read/Count exercises. And of course, this was Adibou's first appearance! Although the floppy release has him speak the same strange gibberish as heard in the [Goblins series](https://wiki.scummvm.org/index.php/Goblins_series), he is still the patient and funny learning partner that he's remembered for being in later games.
```

ScummVM supports all of the game's releases, from the initial 16-color floppy ones to later 1994 CD versions with richer graphics and sound effects. So dig out your dusty Adibou disks, get a [ScummVM daily development build](https://www.scummvm.org/downloads/#daily), and start collecting sweets while completing math and reading exercises! Please also [contact us](https://www.scummvm.org/contact/) if you own a version ScummVM is not yet aware of, and report to the [issue tracker](https://bugs.scummvm.org/) any problem you may encounter.

> ----

Reviewer: Codacy is failing because it says we should use only three hyphens instead of four.
```suggestion
---
```

---

> title: A new World of Discoveries

Reviewer: The headlines for the website are written in [title case](https://en.wikipedia.org/wiki/Title_case), which generally capitalizes all words except for certain words like articles and prepositions.
```suggestion
title: A New World of Discoveries
```

author: sdelamarre
date: 1689415200
---

> _Adibou_, first of the name, is now ready for public testing!

Reviewer: "First of the name" makes it sound like regnal succession or something (King Adibou I, succeeded by King Adibou II). What about this?

```suggestion
_Adibou_, the first game in the beloved children's series, is now ready for public testing!
```

Author: Agreed - I wasn't so sure about using that expression in that context (I do like the idea of a "King Adibou II", though :grin:)

> Also known as _Adi Jr.'s World of Discoveries_, this 1991 Coktel Vision title pioneered the Adibou series, an educational suite aimed at children of age 4 to 7 years.

Reviewer: I don't think that "pioneered" applies to something being the first of a line of intellectual property. We could say that Sierra pioneered graphical adventure games with *King's Quest I*, but we couldn't say that it pioneered the *King's Quest* franchise.

Maybe we can use a different word to say that this was the origin of the franchise.

```suggestion
Also known as _Adi Jr.'s World of Discoveries_, this 1991 Coktel Vision title began the Adibou series, an educational suite aimed at children of age 4 to 7 years.
```

I also don't think that "suite" is the correct word to use here. [According to Wikipedia](https://en.wikipedia.org/wiki/Software_suite), a software suite is a collection of software sharing a similar user interface and intended to exchange data (e.g. Microsoft Office, Adobe Creative Cloud). The word "series" does not appear to b a synonym for "suite". Since the *Adibou* series doesn't match this criteria, I think we need a different word.

```suggestion
Also known as _Adi Jr.'s World of Discoveries_, this 1991 Coktel Vision title pioneered the educational Adibou series aimed at children of age 4 to 7 years.
```

Author: Yes, I like the verb "to pioneer", but also had doubts about using it in that context, "began" is enough and better. Same for "suite", thanks for the explanation about the nuance.

> Many of the iconic activities and mechanisms of the series were already part of this first opus: planting flowers, breakout game, sweets offered as rewards upon completing Read/Count exercises... And of course the main character, Adibou, proving himself as a patient and funny learning partner, although he could only speak goblin gibberish at the beginning!

Reviewer: The first sentence doesn't follow [parallelism](https://en.wikipedia.org/wiki/Parallelism_(rhetoric)). Note how "_planting_" is an action, but "breakout game" is not (but "_playing_ the game "Breakout" would be). Also, I'm assuming that "breakout game" means [this](https://en.wikipedia.org/wiki/Breakout_(video_game)), so I'm capitalizing it.

```suggestion
Many of the iconic activities and mechanisms of the series were already part of this first opus: planting flowers, playing the game *Breakout*, and being rewarded with sweets upon completing Read/Count exercises... And of course the main character, Adibou, proving himself as a patient and funny learning partner, although he could only speak goblin gibberish at the beginning!
```

The sentence is pretty long with the second half having lots of commas. I think it would be better to break it into two. I'm also not understanding the part about speaking goblin gibberish, so I tried to clarify it based on my best guess.

```suggestion
Many of the iconic activities and mechanisms of the series were already part of this first opus: planting flowers, breakout game, sweets offered as rewards upon completing Read/Count exercises. And of course, this was Adibou's first appearance! Although he could only speak goblin gibberish in this game, he is still the patient and funny learning partner that he's remembered for being in later games.
```

Author: I agree on your remark about parallelism, but wouldn't the sentence be a bit lighter using substantives everywhere instead of "actions" ? E.g. fixing parallelism replacing "planting flowers" by "flowers gardening", and leaving the rest of the sentence as is?

Author: On the second part, I agree the sentence is not digest and needs to be split. The "goblin gibberish" part was referring to the fact the first floppy disk versions were non-talky and used instead the same funny "gibberish dialect" as in the Gobliiins games - I'll try to rephrase it more clearly.

Reviewer: > E.g. fixing parallelism by replacing "planting flowers" by "flowers planting", and leaving the rest of the sentence as is?

That could work too, but "flower gardening" instead of "flowers gardening" is what sounds right to me as an (American) native speaker. I think that's because "flower" functions as a descriptor and descriptors aren't usually plural nouns.

For some reason, "breakout game" still sounds off to me if we put in "flower gardening" because it doesn't sound like an action (technically it's a [gerund](https://en.wikipedia.org/wiki/Gerund)). Imagine if I said: "we have three activities: flower gardening, ScummVM, and sweets". The last two aren't actions but the first one is.

I think we need to make it either something like "many of the iconic activities […]: flower gardening, playing breakout (capitalized?), and receiving sweets". Or we could make them not actions: "many of the iconic elements: flowers, the breakout game [or just "breakout"], and sweets".

> The title received substantial updates along the path, from the initial 16-colors floppy releases to later 1994 CD versions with richer graphics and sound effects. All of them should work on ScummVM. So dig out your dusty Adibou floppies or CDs, get a [ScummVM daily development build](https://www.scummvm.org/downloads/#daily), and start collecting sweets while solving maths and reading exercises! Please also [contact us](https://www.scummvm.org/contact/) if you own a version ScummVM is not yet aware of, and report to the [issue tracker](https://bugs.scummvm.org/) any problem you may encounter.

Reviewer: "Along the path" is not a phrase I've ever heard as an (American) native English speaker. There is the phrase "along the way", but even then, I think there are better way to say this. I think we can be more confident than saying they *should* work in ScummVM 😄 And it's redundant to say "floppy" and "CD" in two sentences.

```suggestion
ScummVM supports all of the game's releases, from the initial 16-color floppy release to later 1994 CD version with richer graphics and sound effects. So dig out your dusty Adibou disks, get a [ScummVM daily development build](https://www.scummvm.org/downloads/#daily), and start collecting sweets while solving maths and reading exercises! Please also [contact us](https://www.scummvm.org/contact/) if you own a version ScummVM is not yet aware of, and report to the [issue tracker](https://bugs.scummvm.org/) any problem you may encounter.
```

I think "solving maths" is a British phrase, so I'm leaving that alone (Americans like me would say "solving math problems"). However, you have "solving maths" (action, object) and "reading exercises" (descriptor, object), which again violates parallelism.

Perhaps we can fix both the parallelism and the extra use of a locale-specific phrase.

```suggestion
The title received substantial updates along the path, from the initial 16-colors floppy releases to later 1994 CD versions with richer graphics and sound effects. All of them should work on ScummVM. So dig out your dusty Adibou floppies or CDs, get a [ScummVM daily development build](https://www.scummvm.org/downloads/#daily), and start collecting sweets while completing math and reading exercises! Please also [contact us](https://www.scummvm.org/contact/) if you own a version ScummVM is not yet aware of, and report to the [issue tracker](https://bugs.scummvm.org/) any problem you may encounter.
```

Author: On the "should support", it may seem overly careful, however, a few months ago, while I thought I was over with Adibou1 work, when a user (@BJNFNE) came with a localized CD version including many improvements and using engine features not yet implemented in ScummVM. But yes, let's assume is will not happen again. :laughing:

I fully agree on the other remarks, I'll take you version as-is.

Reviewer: What I was thinking is that we are confident that we support all known versions, and I thought that the original phrase sounded like we had less confidence that we supported all known versions. We do request info about new versions at the end, which I think is all we need.

> 
> _Caveat: no candy parcel support_
> 
> When hardworking children complete their virtual sweets collection, a message appears on the screen, inviting them to send a copy of the savegame on a floppy disk to Coktel Vision by post. At the time the game was released, one would receive in return… a parcel containing Coktel goodies and actual sweets!
> 
> Unfortunately, ScummVM will not be able to revive this beautiful service. So if you happen to show _Adibou_ to younger players, be prepared to fetch by yourself a candy pack from the nearest shop, when they finish the game!

Reviewer: This seems like an odd thing to include: there are other games ScummVM supports that encouraged real-world activity that is no longer possible (like the *Ultima* games saying to call a phone number to inform them of their success, resulting in a certificate being mailed). I don't think anybody would expect ScummVM to support something that involved mailing a disk to the creators. Maybe it's supposed to be a playful joke, but a someone who has never played the game, it didn't land for me.

Personally, I would just remove the whole thing.

```suggestion
```

But if really want to keep it, I think it would be better presented if it were divided by the announcement and presented more like this.

```suggestion

----

_As much as ScummVM wants to help replicate the original gaming experience, there is one feature we are unfortunately unable to support. When hardworking children complete their virtual sweets collection, a message invites them to mail a floppy disk with their savegame to Coktel Vision, who would send them a parcel containing Coktel goodies and actual sweets!

So if you are introducing children to _Adibou_, be prepared to go to the nearest shop and buy some candy when they finish the game!
```

Author: Yes, the last "Caveat" part was indeed kind of a joke (I also think no one would expect us to prepare and send candy parcels by post! :laughing:), but maybe not obvious enough. 

It may be only nostlagia speaking (as I received myself such a parcel as a child, and keep a vivid memory of that! :smile:), but I'd like to keep that last part though, as that detail about the end-game is no so well-known, even among players familiar with Adibou. And I really  I like your rewriting, which is very close to what I intended to express!

Reviewer: Great! After thinking it over, I do see why this would be meaningful, especially to those who grew up with it, and I support keeping it. I think my hangup was that I didn't have the context and it took several sentences before understanding it. I also was still in "technical mode" because that was the previous paragraph and I wasn't quite switching to "joke mode". I think the dividing line will help.
